bluepyentity/app/explorer.py

```diff
@@ -176,6 +178,8 @@ def hide_hints(self):
             for elem in elems:
                 # +2 because of the []
                 elem.right_crop(self._size_combination + 2)
+        self._cur_kt = self._initial_tree
```

Comment:
`self._initial_tree` is just an empty `_tree()`, right?

Reply:
initially it is empty, then it is filled in with the keys structure.

Reply:
`self._initial_tree` is only assigned to in the `_init_state()`, right after `self._cur_kt = _tree()`, so maybe it's missing an update somewhere?

Reply:
it is updated by "add_to_keytree". It is a reference to the root of the tree.